pipelines/scripts/private_endpoint/private_endpoint_manager.py

```diff
@@ -0,0 +1,110 @@
+from typing import List, Dict, Any, Iterable
+from abc import ABC, abstractmethod
+from pipelines.scripts.util import Util
+import logging
+import json
+
+
+class PrivateEndpointManager(ABC):
+
+    def get(self, private_endpoint_id: str) -> Dict[str, Any]:
+        """
+            Return the details of the given private endpoint
+
+            :param private_endpoint_id: The private endpoint to fetch the details for
+        """
+        command_to_run = (
+            "az network private-endpoint-connection show"
```

Comment:
These implicitly return JSON, but to ensure consistency across all environments maybe you can add (--output json) to every CLI command ?

Reply:
I'll add this in